tasks/build.py

```diff
@@ -440,7 +441,15 @@ def check_build_permission(pr, event_info):
     build_labeler = event_info['raw_request_body']['sender']['login']
     if build_labeler not in build_permission.split():
         log(f"{funcname}(): GH account '{build_labeler}' is not authorized to build")
-        # TODO update PR comments for this bot instance?
+        no_build_permission_comment = buildenv.get(NO_BUILD_PERMISSION_COMMENT)
+        gh = github.get_instance()
+        repo_name = event_info["raw_request_body"]["repository"]["full_name"]
+        repo = gh.get_repo(repo_name)
+        pull_request = repo.get_pull(pr.number)
+        pull_request.create_issue_comment(
+            no_build_permission_comment.format(build_labeler=build_labeler,
```

Comment:
This code is almost identical to what we have for `deploy`, should this be factored out to a function used in both?

Reply:
Yeah, can be done in some code polishing/refactoring.

Reply:
Created issue #180 for this.